Document/0x03-Overview.md

```diff
@@ -1,75 +1,82 @@
 
-![Document Overview](Images/Chapters/0x03/owasp-mobile-overview.jpg)
+- The **Mobile Security Testing Guide (MSTG):** A manual for testing the security of mobile apps. It provides verification instructions for MASVS requirements and operating system-specific best practices (currently for Android and iOS). The MSTG helps ensure completeness and consistency of mobile app security testing. It can also be a standalone learning resource and reference guide for mobile app security testers.
+
+- The **Mobile App Security Checklist:** A checklist for tracking compliance against the MASVS during practical assessments. We simplify mobile app penetration testing by linking each requirement in the checklist to the corresponding MSTG test case.
 
-For example, the MASVS requirements could be used in the planning and architecture design stages, while the checklist and testing guide may serve as a baseline for manual security testing or as a template for automated security tests during or after development. In the next chapter, we'll describe how the checklist and guide can be practically applied during a mobile application penetration test.
+![Document Overview](Images/Chapters/0x03/owasp-mobile-overview.jpg)
 
-## Organization of the Mobile Security Testing Guide
+For example, the MASVS requirements could be used in an app's planning and architecture design stages while the checklist and testing guide may serve as a baseline for manual security testing or as a template for automated security tests during or after development. In the next chapter *(Q- what chapter? let's name it)*, we'll describe how you can apply the checklist and MSTG to a mobile app penetration test.
 
-All requirements specified in the MASVS are described in technical detail in the testing guide. The main sections of the MSTG are explained briefly in this chapter.
+## Navigating the Mobile Security Testing Guide 
 
-The guide is organized as follows:
+The MSTG contains descriptions of all requirements specified in the MASVS. The MSTG contains the following main sections:
 
-- In the general testing guide (the following few chapters), we present the mobile app security testing methodology, and talk about general vulnerability analysis techniques as they apply to mobile application security.
+1. The **General Testing Guide** contains mobile app security testing methodology and general vulnerability analysis techniques as they apply to mobile app security.
+> add "chapters x to y"? Or give links? Or both?
 
-- The Android Testing Guide covers everything specific to the Android platform, including security basics, security test cases, and reverse engineering and tampering techniques and preventions.
+2. The **Android Testing Guide** covers mobile security testing for the Android platform, including security basics, security test cases, reverse engineering techniques and preventions, and tampering techniques and preventions.
```

Comment:
It's obvious what it means, but I am not sure whether "preventions" is a word. Maybe just replace it with "prevention"?